server/events/yaml/raw/global_cfg.go

```diff
@@ -163,6 +179,13 @@ func (r Repo) ToValid(workflows map[string]valid.Workflow) valid.Repo {
 		id = r.ID
 	}
 
+	var branchRegex *regexp.Regexp
+	if r.HasRegexBranch() {
+		withoutSlashes := r.Branch[1 : len(r.Branch)-1]
+		// checked in repo Validate()
+		branchRegex = regexp.MustCompile(withoutSlashes)
+	}
```

Comment:
Consider allowing specify single branch using plain `branch-name` syntax, similar to non-regex for the `id`.

Reply:
I considered this. `id` allows a string or regex, but it feels kinda cumbersome. The default value needs to continue matching all branches. I favored `branch` field always being a regex (default `/.*/`) (and if you want to match a single name, you could just set it to `/master/`) so we don't need to condition on which sort of string it is.

If a maintainer prefers the opposite, I'm happy to refactor.

Reply:
ok